ang/civicase-base.ang.php

```diff
@@ -51,6 +51,7 @@
 function expose_settings(array &$options, array $defaults) {
   $options['allowMultipleCaseClients'] = (bool) Civi::settings()->get('civicaseAllowMultipleClients');
   $options['allowCaseLocks'] = (bool) Civi::settings()->get('civicaseAllowCaseLocks');
+  $options['caseTypeCategoriesWithAccessToActivities'] = CRM_Civicase_Helper_CaseCategory::getCaseTypeCategoriesWithAccessToActivities();
```

Comment:
In my opinion,
1. `API call` - Dont think its worth it, also in which api would we add it?
2. `conditionally add it`- Not much sure of this too, because we are treating this as a global value, which can be used by anyone, hence it was added to the `CaseTypeCategoryProvider`. So it should always be accessible. 

@tunbola opinion?

Reply:
Yeah. Don't think an API call is worth it either. 
If this will be only useful to one screen for now, it might make sense to add it conditionally.

Something like:
```php
if (in_array(CRM_Utils_System::currentPath(), ['civicrm/contact/view', 'civicrm/case/contact-case-tab'])) {
    $options['caseTypeCategoriesWithAccessToActivities'] = CRM_Civicase_Helper_CaseCategory::getCaseTypeCategoriesWithAccessToActivities();
}
```

Reply:
@reneolivo @tunbola I feel that we are overcomplicating it, because the generic settings should be available for all pages, because they are settings. 

`caseTypeCategoriesWithAccessToActivities` is a generic name, and can be used by any pages if needed. Right now its not needed by other pages I agree, but if we start limiting settings to specific pages, not sure how good that will be. 

For example `caseActions` is only used by manage case page, should we then hide that from other pages? or what about `allowCaseLocks`? there are many more examples like that.

Maybe lets discuss this on Monday, before having a final decision.